proxy/http2/Http2ConnectionState.h

```diff
@@ -38,7 +38,7 @@ enum Http2SendADataFrameResult {
   HTTP2_SEND_A_DATA_FRAME_DONE       = 3,
 };
 
-enum Http2ShutdownState { NOT_INITIATED, INITIATED, IN_PROGRESS };
+enum Http2ShutdownState { NOT_PLANNED, NOT_INITIATED, INITIATED, IN_PROGRESS };
```

Comment:
Looking at the code for handling `HTTP2_SESSION_EVENT_SHUTDOWN_INIT`, is it safe to schedule the `HTTP2_SESSION_EVENT_SHUTDOWN_CONT` in 2 seconds? What guarantees that the `Http2ConnectionState` is still alive at that time?